### PR TITLE
[QA] SISRP-35757 - Upgrades JRuby to 9.1.5.0 (take 3)

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.berkeley.edu/browse/SISRP-35757

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -119,7 +121,7 @@ group :development, :test , :testext do
   gem 'rspec-core', '~> 3.1.7'
   gem 'rspec-rails', '~> 3.1.0'
   gem 'rspec-mocks', '~> 3.1.3'
-  gem 'rspec-support', '~> 3.1.2'
+  gem 'rspec-support', '~> 3.1.0'
```
Comment (davidlimcheng) [inline Gemfile]: @lyttam Why do we have to downgrade this gem?

**Comment by davidlimcheng** [comment]
Thanks @lyttam.  We're holding off on merging this until we get the green light from the Ops team, right?

**Comment by lyttam** [comment]
@davidlimcheng it can be merged anytime, but we won't deploy until after Ops does their work.

File: Gemfile (hunk 1 above)
Comment (lyttam) [reply Gemfile]: @davidlimcheng It was causing conflicts before, though that may have only been the case under the newer bundler version.

**Comment by davidlimcheng** [comment]
Thanks @lyttam.  Since the testext build will fail if we merge before upgrading the "Docker" server, I'll hold off on merging this until we get our Ops team back.

**Comment by davidlimcheng** [comment]
Looks like we're ready to merge!

Note:  @lyttam has added the configuration change to the JVM_OPTS that raises the threshold for OldGenHeap size that kicks off a full GC cycle.

**Pull request merged**